lib/Video/Adapter/Ffmpeg.php

```diff
@@ -262,14 +262,16 @@ public function getDuration()
         $output = $this->getVideoInfo();
 
         // get total video duration
-        preg_match("/Duration: ([0-9:\.]+),/", $output, $matches);
-        $durationRaw = $matches[1];
-        $durationParts = explode(':', $durationRaw);
+        $result = preg_match("/\s+Duration: ((\d\d):(\d\d):(\d\d)\.(\d+))/", $output, $matches);
```

Comment:
So the milliseconds should be in the return value again
```suggestion
        $result = preg_match("/\s+Duration: ((\d\d):(\d\d):(\d\d\.\d+))/", $output, $matches);
```